support/docker/production/config/custom-environment-variables.yaml

```diff
@@ -101,9 +101,13 @@ transcoding:
     1080:
       __name: "PEERTUBE_TRANSCODING_1080P"
       __format: "json"
-    
 
 instance:
   name: "PEERTUBE_INSTANCE_NAME"
   description: "PEERTUBE_INSTANCE_DESCRIPTION"
   terms: "PEERTUBE_INSTANCE_TERMS"
+
+services:
+  csp-logger:
+    __name: "PEERTUBE_SERVICES_CSPLOGGER"
+    __format: "json"
```

Comment:
isn't a string?

Reply:
I don't know, the syntax isn't documented anywhere.

Reply:
Is this the endpoint for the [csp-logger](https://github.com/mozilla/csp-logger)? In that case it should be an URL, no?

Reply:
Maybe I'm misunderstanding how this file works, but it seems like json is the wrong type for an URL.

Reply:
https://github.com/lorenwest/node-config/wiki/Environment-Variables#custom-environment-variables